packages/python-packages/apiview-stub-generator/setup.py

```diff
@@ -34,7 +34,8 @@
     install_requires=[
         "charset-normalizer",
         "pylint",
-        "azure-pylint-guidelines-checker"
+        "azure-pylint-guidelines-checker",
+        "pkginfo"
```

Comment:
pkginfo is currently used by ci_tools: https://github.com/Azure/azure-sdk-for-python/blob/21ff3945ce37cf36758563637c6e8a15101b1989/eng/ci_tools.txt#L10

Reply:
@scbedd - Do you see any issues with this?

Alternatives I've tried:
- importlib.metadata --> Requires that the package is installed first to grab metadata. In the case of something like azure-quantum that fails to install without the optional deps, this won't work.
- build --> Tried a few things but didn't seem as clean as pkginfo. I can go back and test this out more if pkginfo doesn't work.